ibis/backends/tests/test_temporal.py

```diff
@@ -317,6 +331,9 @@ def test_timestamp_extract_milliseconds(backend, alltypes, df):
     pyspark=["pandas<2.1"],
     reason="test was adjusted to work with pandas 2.1 output; pyspark doesn't support pandas 2",
 )
+@pytest.mark.broken(
+    ["flink"], raises=AssertionError, reason="numpy array are different"
+)
```

Comment:
@mfatihaktas I went ahead and removed the extra-long error message, as per Gil's comment. I'm curious to understand why this test can't be made to work, though? I saw the translation is not including the string argument to provide timestamp format; do you think that could be related?

Reply:
> I went ahead and removed the extra-long error message, as per Gil's comment.

Thanks.

> I saw the translation is not including the string argument to provide timestamp format; do you think that could be related?

Sorry, I could not understand what you mean by "string argument to provide timestamp format". I also cannot find which test function this comment was for; it is outdated and I could not locate the relevant function.

Reply:
@deepyaman Thanks for the details. In my opinion, we can mark this with `TODO` and work on it during our second iteration of the tests.